### PR TITLE
Fix empty suffix on tag

### PR DESCRIPTION
**Pull request by Nenkai**

### CODE REVIEW

File: runtime/commands/taglogic.js [hunk 1]
```diff
@@ -14,6 +14,7 @@ Commands.tag = {
   aliases: ['t'],
   fn: function (msg, suffix, bot) {
     var index = suffix.split(' ')
+    if (!suffix) {
```
Comment (SteamingMutt) [inline runtime/commands/taglogic.js]: Please indent

File: runtime/commands/taglogic.js [hunk 2]
```diff
@@ -14,141 +14,141 @@ Commands.tag = {
...
-      var content = index.slice(2, index.length).join(' ')
-      db.find({
-        _id: index[1].toLowerCase()
-      }, function (err, res) {
-        if (err) {
-          msg.channel.sendMessage('Something went wrong.')
-        } else if (res) {
-          if (res.length > 0) {
-            msg.channel.sendMessage('A tag with that name already exists.')
+	  if (!suffix) {
```
Comment (SteamingMutt) [inline runtime/commands/taglogic.js]: Please indent *correctly*

File: runtime/commands/taglogic.js [hunk 3]
```diff
@@ -14,141 +14,141 @@ Commands.tag = {
...
+              msg.channel.sendMessage('That tag does not exist.')
+              return
+            } else {
+              msg.channel.sendMessage(res[0].content.replace('@everyone', '@every\u200Bone').replace('@here', '@he\u200Bre'))
+            }
+          }
+        })
+      }
+  	} else {
+		  msg.channel.sendMessage('No arguments entered.')
```
Comment (SteamingMutt) [inline runtime/commands/taglogic.js]: smh

File: runtime/commands/taglogic.js (hunk 2 above)
Comment (Nenkai) [inline runtime/commands/taglogic.js]: :+1:

**Pull request merged**